### PR TITLE
feat/MSSDK-1957: Pause Billing - updating My Account Copy

### PR DESCRIPTION
**Pull request by m4tewoosh**
### Description

Added specific copy for paused offer with new date format

### CODE REVIEW

File: src/styles/variables.js [hunk 1]
```diff
@@ -71,7 +71,7 @@ export const TinyFont = '11px';
 export const MicroFont = '9px';
 
 export const BoldFont = '700';
-export const MediumFontWeight = '600';
+export const MediumFontWeight = '500';
```
Comment (Saddage) [inline src/styles/variables.js:74]: why was this changed? will this be consistent with our UX designs across all the components and views?

File: src/util/planHelper.ts [hunk 2]
```diff
@@ -313,4 +313,14 @@ export const dateFormat = (timestamp: number, showTime = false) => {
   return resultString;
 };
 
+export const textMonthDateFormat = (timestamp: number): string => {
+  const date = new Date(timestamp * 1000);
+
+  const month = date.toLocaleString(undefined, { month: 'long' });
```
Comment (Saddage) [inline src/util/planHelper.ts:319]: why `undefined`? we should format this according to the user's locale

Comment (m4tewoosh) [reply src/util/planHelper.ts:319]: And that's what `undefined` does here, the user's locale is taken.

"If you omit this parameter, the default locale of the JavaScript runtime is used" - from the docs

File: src/styles/variables.js (hunk 1 above)
Comment (m4tewoosh) [reply src/styles/variables.js:74]: Changed back to 600, renamed to `SemiBoldFont`

**Review by pawelacio — APPROVED** [review]

**Review by pawelacio — APPROVED** [review]
